perllib/FixMyStreet/App/Controller/Report.pm

```diff
@@ -335,8 +337,21 @@ sub inspect : Private {
         my %update_params = ();
 
         if ($permissions->{report_inspect}) {
-            foreach (qw/detailed_information traffic_information/) {
-                $problem->set_extra_metadata( $_ => $c->get_param($_) );
+            $problem->set_extra_metadata( traffic_information => $c->get_param('traffic_information') );
+
+            if ( my $info = $c->get_param('detailed_information') ) {
+                $problem->set_extra_metadata( detailed_information => $info );
+                if ($c->cobrand->max_detailed_info_length &&
+                    length($info) > $c->cobrand->max_detailed_info_length
+                ) {
+                    $valid = 0;
+                    $c->stash->{errors} ||= [];
```

Comment:
Doesn't matte, and I see this is same as elsewhere, but just to note this line isn't needed as it will be auto-vivified by the next line.